src/SFA.DAS.Reservations.Web/Views/Error/AccessDenied.cshtml

```diff
@@ -6,25 +6,30 @@
 
         <div class="govuk-grid-column-two-thirds">
 
-            <h1 class="govuk-heading-xl">Access denied</h1>
-
-
             @if (Configuration["AuthType"].Equals("provider", StringComparison.CurrentCultureIgnoreCase))
             {
-                <p class="govuk-body">You don’t have permission to access the Apprenticeship Service.</p>
-                <h2 class="govuk-heading-m">Help</h2>
-                <p class="govuk-body">You can contact the Apprenticeship Service for advice or help on how to use the service.</p>
-                <h3 class="govuk-heading-m">Apprenticeship Service</h3>
-                <p class="govuk-body">Telephone: 0800 015 0600 <a href="https://www.gov.uk/call-charges" target="_blank">Find out about call charges</a></p>
+                <h1 class="govuk-heading-xl">
+                    You do not have permission to access this page
+                </h1>
+
+                <p class="gov-body">
+                    If you think you should have permission to access this page or want to confirm what permissions you have, contact an account owner.
+                </p>
+
+                <br>
+
+                <p class="gov-body govuk-!-margin-bottom-8">
+                    <a asp-external-controller="service" asp-external-action="accounts">Go back to home page</a>
```

Comment:
this ```asp-external-controller="service" asp-external-action="accounts"``` won't work for the provider site.
```asp-external-controller=""``` would be better. This should redirect back to the route of the provider site/homepage.

Like here: https://github.com/SkillsFundingAgency/das-reservations/blob/943cbb77e72636c05dead993fa6564787b23af43/src/SFA.DAS.Reservations.Web/Views/Shared/_Layout.cshtml#L102

Reply:
Update